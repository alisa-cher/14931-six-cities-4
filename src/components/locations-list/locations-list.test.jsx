import React from 'react';
import LocationsList from "./locations-list.jsx";
import renderer from 'react-test-renderer';

const locations = [{
  location: {
    latitude: 1,
    longitude: 2,
    zoom: 3
  },
  name: `Brussels`
}];

it(`Locations List renders correctly`, () => {
  const tree = renderer
    .create(
        <LocationsList
          locations={locations}
          activeLocation={locations[0]}
          onMenuClick={() => {}}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

