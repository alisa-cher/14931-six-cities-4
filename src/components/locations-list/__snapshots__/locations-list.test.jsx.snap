// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Locations List renders correctly 1`] = `
<div
  className="tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="locations__list tabs__list"
    >
      <li
        className="locations__item"
      >
        <a
          className="tabs__item--active locations__item-link tabs__item"
          onClick={[Function]}
        >
          <span>
            Brussels
          </span>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
